### DISCUSSION TITLE
默认的IP库不支持ipv6，能支持下ip2location-lite么？

### DISCUSSION THREAD

**Discussion by dianso** [ArtalkJS/Artalk#782] [General]
能否支持

https://lite.ip2location.com/ip2location-lite

这个是免费，每个月更新1次，注册账号即可下载，ipv6数据同时包含ipv4

**Comment by qwqcode** [comment]
但这是英文数据，有中文的支持ipv6的数据库吗？

<img width="593" alt="image" src="https://github.com/ArtalkJS/Artalk/assets/22412567/6f2ef6b8-3354-4f44-a095-2d72bfda8214">

**Reply by clover1420** [reply]
@qwqcode 
https://github.com/nmgliangwei/qqwry.ipdb 这有一个哎
